app/notifications/process_client_response.py

```diff
@@ -83,5 +84,7 @@ def process_sms_client_response(status, reference, client_name):
 
     create_outcome_notification_statistic_tasks(notification)
 
+    send_delivery_status_to_service.apply_async([notification.id], queue=QueueNames.NOTIFY)
```

Comment:
Need to wrap notification_id with str(notification_id) otherwise there will be a serialisation error

Reply:
changed in 3594fc.